core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -62,10 +62,16 @@
  * <LI><b>readmodifywriteproportion</b>: what proportion of operations should be read a record,
  * modify it, write it back (default: 0)
  * <LI><b>requestdistribution</b>: what distribution should be used to select the records to operate
- * on - uniform, zipfian, hotspot, ordered or latest (default: uniform)
+ * on - uniform, zipfian, hotspot, ordered, exponential or latest (default: uniform)
```

Comment:
Yes, I've noticed in the code there's also an exponential distribution. Not my doing